src/pudl/transform/ferc1.py

```diff
@@ -4593,7 +4644,7 @@ def convert_xbrl_metadata_json_to_df(
             {
                 "name": "electric_plant_in_service_and_completed_construction_not_classified_electric",
                 "weight": 1.0,
-                "source_tables": ["plant_in_service_ferc1"],
+                "source_tables": ["core_ferc1__yearly_plant_in_service_sched204"],
                 "utility_type": "electric",
             }
```

Comment:
this old table name was resulting in the vast majority of the $s from this table not showing up in the rate base table